### PR TITLE
Add symbolic inverse trig support

### PR DESCRIPTION
**Pull request by CrashAndSideburns**
PrairieLearn doesn't support the inverse trigonometric functions arcsin, arccos, and arctan. This adds support for their use in symbolic answers, which seems important as their absence renders a great deal of calculus problems typical of first year unanswerable. arcsin, arccos, and arctan were chosen for the names despite asin, acos, and atan being used by SymPy, as I think the "arc-" prefix is more common and familiar to students. If any further justification is necessary, the "arc-" prefix is also specified by ISO 80000-2.

### CODE REVIEW

**Comment by CrashAndSideburns** [comment]
I have reviewed and hereby sign the CLA

**Comment by mwest1066** [comment]
I think the new operators also need to be added here: https://github.com/PrairieLearn/PrairieLearn/blob/59ea060f619150086dad5a5e35dd74941c7a9353/elements/pl-symbolic-input/pl-symbolic-input.py#L62

**Comment by mwest1066** [comment]
> arcsin, arccos, and arctan were chosen for the names despite asin, acos, and atan being used by SymPy, as I think the "arc-" prefix is more common and familiar to students.

I'm fine to add these as `arcsin`, etc, but maybe we should also add the `asin` variants so that anyone who is familiar with python/C/matlab/julia/etc will also be able to use their expected names? I don't think there is any harm in adding the functions twice with two different names (`arcsin`/`asin`)?

**Comment by CrashAndSideburns** [comment]
Added `acos`, `asin`, and `atan` to `_Constants.functions`, so they should be accessible under either name. I'm a bit unsure as to the appropriate course of action with regard to adding them to the list of operators. I've tentatively added all of the names to the list, but I'm not sure if something like `acos/arccos` would be a more sensible way of writing it than having both `acos` and `arccos` in the list, since they both refer to the same function and the presence of both might confuse the end user.

File: lib/python_helper_sympy.py [hunk 1]
```diff
@@ -28,6 +28,9 @@ def __init__(self):
             'cos': sympy.cos,
             'sin': sympy.sin,
             'tan': sympy.tan,
+            'arccos': sympy.acos,
+            'arcsin': sympy.asin,
+            'arctan': sympy.atan,
```
[the remark this answers is not in the document]
Comment (CrashAndSideburns) [reply lib/python_helper_sympy.py:33]: Added `atan2` as `arctan2` and `atan2` in 6c257ca06c06. Also added them both to the operators list in the same way as with the other trig functions in dc6df27b596e29c35bbf801b605fd294d2db41ed, so my potential reservations about having both names independently in the operators list applies here too.

**Pull request merged**